practice-app/server/src/routes/chess/chess.controller.js

```diff
@@ -49,6 +51,20 @@ const ChessController = {
     makeMove: async function (req, res) {
         // Get user input
         const { gameId, moveStr } = req.body;
+        const user = req.auth;
+
+        const game = (
+            await ChessGame.find(
```

Comment:
Can you use ChessGame.findOne instead of ChessGame.find so that you do not have to get element by [0]